projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -70,11 +70,12 @@ function Price( { value, currency, isOld } ) {
  *
  * @param {object} props                    - Component props.
  * @param {string} props.slug               - Product slug
+ * @param {Function} props.onClick          - Callback for Call To Action button click
```

Comment:
minor: `onClick` name is confusing for the `<ProductDetail />`. Maybe `onAdd()` is more descriptive.